src/test/java/org/wildfly/security/ldap/X509EvidenceVerificationSuiteChild.java

```diff
@@ -44,6 +49,19 @@
  */
 public class X509EvidenceVerificationSuiteChild {
 
+    private static final String LDAP_DIRECTORY_LOCATION = "./target/test-classes/ldap";
+    private static final String LDIF_LOCATION = "/elytron-x509-verification.ldif";
+
+    @AfterClass
+    public static void cleanUpLdapFiles() throws Exception {
+        File workingDirLDIF = new File(LDAP_DIRECTORY_LOCATION);
+        if (workingDirLDIF.exists() == false) {
+            workingDirLDIF.mkdirs();
```

Comment:
If the directory doesn't exist, the .bak file won't exist so this if statement can be removed and replaced with a check to make sure the directory does exist.

Reply:
Fixed